src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -676,6 +676,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 		UpdateAndroidResources;
 		$(ApplicationResolveReferencesDependsOn);
 	</ResolveReferencesDependsOn>
+	<DeferredBuildDependsOn>$(DeferredBuildDependsOn);UpdateAndroidResources</DeferredBuildDependsOn>
```

Comment:
I think I'm missing something but...  shouldn't `$(DeferredBuildDependsOn)` be used somewhere?  Presumably as a value to a `DependsOnTargets` attribute?

Reply:
It's used in the IDE that supports the concept of deferred builds (currently, just VS): https://github.com/xamarin/XamarinVS/blob/master/src/MSBuild/Xamarin/Xamarin.Sdk.targets#L81